src/components/dictionaryConcepts/containers/SelectAnswers.jsx

```diff
@@ -13,8 +13,24 @@ class SelectAnswers extends Component {
     inputValue: '',
     options: [],
     isVisible: false,
+    isClicked: false,
   }
 
+  componentDidMount() {
+    this.setState({ inputValue: this.props.defaultValue });
+  }
+
+    resetInput = (e) => {
```

Comment:
This approach to deleting the answer introduces a bug. Since both the `Add` and `Edit` Concept share the same `Answer` component, it expects `removeCurrentAnswer` to be passed as a prop, which it is not in the add concept case.
It therefore crashes when one places the cursor in the input box for creating a new answer.
[Recording](https://drive.google.com/file/d/1ISGperfma3cBujGjXAui0WSLo3sB-l83/view?usp=sharing).
Screenshot:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/12200860/56604600-040ac980-660b-11e9-8172-676a246a21f0.png">

Reply:
Thanks for helping discovering this. I'm working on it

Reply:
Fixed